### DISCUSSION TITLE
php Fatal error: Allowed memory size bytes exhausted

### DISCUSSION THREAD

**Discussion by hart323** [phalcon/cphalcon#15355] [Q&A] [answered]
Hi!
I found old threads about the problems with find() in the large mysql tables but did not found the solution how not to load all found data in the memory.

I use phalcon 3.4.
```
$this->view->disable();
echo memory_get_usage(), "<br>", PHP_EOL;
$result = Photo::find();
echo "counter: ", Photo::count(), "<br>", PHP_EOL;
echo memory_get_usage(), "<br>", PHP_EOL;
die();
```

And the result of this is:
```
609472
counter: 601700
**125229824**
```

Any help?
Thanks.

**Comment by Jeckerson** [comment, marked as answer]
Hello

This is not a problem of Phalcon. Default memory limit per request in PHP is 128Mb.

* You can try to increase memory limit `ini_set('memory_limit', '1G') // set memory limit 1Gb per request`
* Or you can try to separate queries by specifying limit
